Modules/Tracker/QuestieTracker.lua

```diff
@@ -1221,103 +1889,100 @@ function QuestieTracker:RemoveQuest(questId)
     end
 end
 
-local hexTable = {
-    "00","11","22","33","44","55","66","77","88","99","AA","BB","CC","DD","EE","FF"
-}
-
-function _QuestieTracker:PrintProgressColor(percent, text)
-    local hexGreen = hexTable[5 + math.floor(percent * 10)]
-    local hexRed = hexTable[8 + math.floor((1 - percent) * 6)]
-    local hexBlue = hexTable[4 + math.floor(percent * 6)]
-
-    return "|cFF"..hexRed..hexGreen..hexBlue..text.."|r"
-end
-
-_RemoveQuestWatch = function(index, isQuestie)
-    Questie:Debug(Questie.DEBUG_DEVELOP, "QuestieTracker: RemoveQuestWatch")
-    if QuestieTracker._disableHooks then
+QuestieTracker.RemoveQuestWatch = function(index, isQuestie)
```

Comment:
This should work, right?

```suggestion
function QuestieTracker.RemoveQuestWatch(index, isQuestie)
```

Reply:
Opps... missed one! I refactored all the other ones. :D